e2e-tests/trailing-slash/cypress/support/utils/trailing-slash.js

```diff
@@ -0,0 +1,14 @@
+export function assetPageVisits(pages) {
+  for (let i = 0; i < pages; i++) {
+    const page = pages[i]
+
+    cy.intercept(new RegExp(`^${page.path}$`), req => {
+      req.continue(res => {
+        expect(res.statusCode).to.equal(page.status)
+        if (page.destiationPage) {
```

Comment:
Should we have an `else` here that asserts `location` is not set?

Reply:
We are also using this util to test for 200s where there's no redirect so there's no else case, only the first `expect` runs.

We could throw if you're passing a 301 without `destinationPage` though